.buildkite/setup_and_upload_artifact.sh

```diff
@@ -32,13 +32,7 @@ fi
 PYTHON_CMD="$PYTHON_PATH .buildkite/upload_artifacts.py"
 echo "Now excuting  upload artifacts script..."
 mkdir -p dist
-buildkite-agent artifact download 'dist/*.pex' dist/
-buildkite-agent artifact download 'dist/*.whl' dist/
-buildkite-agent artifact download 'dist/*.tar.gz' dist/
-buildkite-agent artifact download 'dist/*.deb' dist/
-buildkite-agent artifact download 'dist/*.exe' dist/
-# buildkite-agent artifact download 'dist/*.dmg' dist/
-buildkite-agent artifact download 'dist/*.zip' dist/
+buildkite-agent artifact download 'dist/*' dist/
```

Comment:
What are the disadvantages of doing it this way?

Reply:
Because all I see are advantages

Reply:
I guess it's less explicit. And, because it's less specific, there is a chance you'll have an incomplete list of files to upload (which is kind of the point anyway).